src/v2/codegen/emitter.rs

```diff
@@ -211,6 +211,18 @@ pub trait Emitter: Sized {
         let state = self.state();
         state.reset_internal_fields();
 
+        let m = state.get_meta();
+        if let Some(mut meta) = m.borrow_mut().clone() {
+            if meta.name.is_none() {
+                meta.name = Some(api.clone().info.title)
+            }
+            if meta.version.is_none() {
+                meta.version = Some(api.clone().info.version)
```

Comment:
What's wrong with doing `api.info.version.clone()`? We don't have to clone the entire `api` (it's huge).